webapp/src/components/jira_ticket_tooltip/ticketStyle.scss

```diff
@@ -178,3 +178,57 @@
     border-radius: 2px;
     line-height: 16px;
 }
+
+.fix-version-label {
+    color: #333;
+    margin: 16px 0px;
+    text-align: left;
+    font-family: open sans;
+    font-size: 10px;
+    padding: 0px 0px 2px 0px;
+}
+
+.fix-version-label-value {
+    background-color: rgba(63, 67, 80, 0.08);
+    padding: 1px 8px;
+    font-weight: 600;
+    border-radius: 2px;
+}
+
+.default-style{
+    font-family: sans-serif;
+    font-style: normal;
+    font-weight: 600;
+    font-size: 12px;
+    margin: 10px 0 8px 0;
+    padding: 4px 8px;
+    text-align: center;
+    border-radius: 4px;
+};
+
+.ticket-status{
+    color: #FFFFFF; 
+    background-color: #1C58D9;
+    border-radius: 2px;
+}
+
+.ticket-status-done{
+    color: #FFFFFF;
+    background-color: #3DB887;
+}
+
+.ticket-status-default{
+    color: #3F4350;
+    background-color: rgba(63, 67, 80, 0.16);
+}
+
+.default-avatar{
+    background-color: 'slategrey'; 
```

Comment:
somewhere we have used hash codes, somewhere like this. shouldn't we follow a single approach?